### PR TITLE
OCPQE-19857 Adding normal 4.16 upgrade CI jobs for arm

### PR DESCRIPTION
**Pull request by JianLi-RH**
https://issues.redhat.com/browse/OCPQE-19857

This PR is for arm jobs

/cc @shellyyang1989 @jiajliu @jianlinliu @liangxia @dis016

### CODE REVIEW

**Comment by liangxia** [comment]
If we choose to split the PRs by arch, then perhaps we should include arm64 stable 4.15 -> nightly 4.16 upgrade in this PR, too ?

**Comment by JianLi-RH** [comment]
> If we choose to split the PRs by arch, then perhaps we should include arm64 stable 4.15 -> nightly 4.16 upgrade in this PR, too ?

@liangxia There is no  `arm64 stable 4.15 -> nightly 4.16` introduced, we are using multi instead arm in pr 49710

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  intermediate-arm64:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml:55]: I vaguely remember we had a discussion around this before. The conclusion seems using fast channel as a start version for upgrade jobs.

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-4.16-upgrade-from-stable-4.15-azure-ipi-disk-encryption-set-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-stable-4.16-upgrade-from-stable-4.15-azure-ipi-fullyprivate-proxy-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-4.16-upgrade-from-eus-4.14-aws-ipi-ovn-f28

**Comment by shellyyang1989** [comment]
@lwan-wanglin Please also help review the 4.16 arm job. We will fix the issues if any in the follow-up PRs.

**Comment by lwan-wanglin** [comment]
@shellyyang1989 @JianLi-RH  the same situation for gcp, we can also add several gcp profiles to `openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-stable-4.15.yaml`.
we can pick up some from below profiles in `4.16__multi-nightly `
```
find ./openshift-openshift-tests-private-release-4.16__multi-nightly.yaml | xargs grep "gcp.*arm-f"
- as: gcp-ipi-basecap-none-additionalcaps-arm-f7
- as: gcp-ipi-compact-filestore-csi-arm-f14
- as: gcp-ipi-disk-encryption-arm-f14
- as: gcp-ipi-ovn-mtu-migrate-arm-f28
- as: gcp-ipi-proxy-etcd-encryption-arm-f14
- as: gcp-ipi-proxy-private-arm-f14
- as: gcp-ipi-proxy-tp-arm-f14
- as: gcp-ipi-proxy-oidc-arm-f28
- as: gcp-ipi-sno-etcd-encryption-arm-f14
- as: gcp-ipi-sno-tp-arm-f28
- as: gcp-ipi-user-labels-tags-filestore-csi-tp-arm-f14
- as: gcp-ipi-xpn-mini-perm-byo-hosted-zone-arm-f28
- as: gcp-ipi-xpn-minimal-permission-arm-f28
- as: gcp-ipi-xpn-private-arm-f14
- as: gcp-ipi-xpn-oidc-arm-f28
- as: gcp-upi-private-xpn-ingress-glb-arm-f14
```

one thing need to be noticed, we won't need `arm` keyword in the upgrade profiles

**Comment by JianLi-RH** [comment]
Thanks @lwan-wanglin I will add some of them in a follow up pr.

**Pull request merged**